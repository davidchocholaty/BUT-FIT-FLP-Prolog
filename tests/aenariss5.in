S a S R
S c S R
S c F L
aaacca
