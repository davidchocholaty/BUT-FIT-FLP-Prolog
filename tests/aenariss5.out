Saaacca
aSaacca
aaSacca
aaaScca
aaacSca
aaaFcca
